paddle/fluid/framework/ir/mkldnn/cpu_quantize_squash_pass_tester.cc

```diff
@@ -91,11 +93,11 @@ ProgramDesc BuildConvRequantProgramDesc(bool use_mkldnn, float scale_out,
   }
 
   SetOp(&prog, "conv2d", "Conv1", {"a", "w1", "b1"}, {"d"}, use_mkldnn,
-        {scale_out});
+        {1.23f, scale_out});
   SetOp(&prog, "dequantize", "Dequant", {"d"}, {"e"}, use_mkldnn, {scale1});
   SetOp(&prog, "quantize", "Quant", {"e"}, {"f"}, use_mkldnn, {scale2});
   SetOp(&prog, "conv2d", "Conv2", {"f", "w2", "b2"}, {"i"}, use_mkldnn,
-        {scale_out});
+        {1.23f, scale_out});
```

Comment:
It would be more consistent if the scale of quantize was the same as the input scale of the next op, and the output scale of an operator was the same as the next dequantize op's scale. The other situation is rather unusual if not faulty.
The same applies to the cases below.